src/create-client.ts

```diff
@@ -47,6 +47,8 @@ export const clientConstructor = ({
             error.response.data &&
             error.response.data.description
           ) {
+            // eslint-disable-next-line no-console
+            console.log('error', error.response.data.details)
```

Comment:
Here for the convenience of testing in case the copy request fails to see details. We should probably have a look at errors in #56 rather soon.